Tasks/PublishSymbols/IndexHelpers/DbghelpFunctions.ps1

```diff
@@ -1,26 +1,30 @@
+function Get-DbghelpPath {
+    [CmdletBinding()]
+    param()
+
+    Trace-VstsEnteringInvocation $MyInvocation -Parameter @()
+    $dbghelpPath = [System.IO.Path]::GetFullPath("$PSScriptRoot\..\dbghelp.dll")
+    Assert-VstsPath -LiteralPath $dbghelpPath -PathType Leaf -PassThru
```

Comment:
Updated and added tests to ensure we don't forget any of these binaries in a future archivePackage zip